integration/src/builtin_functions.rs

```diff
@@ -0,0 +1,155 @@
+use std::{collections::HashMap, ops::RangeFrom};
+
+use itertools::Itertools;
+use openvm_stark_backend::p3_field::PrimeField32;
+use wasmparser::{Operator, ValType};
+use womir::loader::{
+    blockless_dag::{BlocklessDag, Operation},
+    flattening::WriteOnceAsm,
+    Module,
+};
+
+use crate::womir_translation::{Directive, OpenVMSettings};
+
+struct BuiltinDefinition {
+    wasm_bytes: &'static [u8],
+    params: &'static [ValType],
+    results: &'static [ValType],
+}
+
+// TODO: add more built-in functions as needed.
+
+const MEMORY_COPY_WASM: BuiltinDefinition = BuiltinDefinition {
```

Comment:
not a big deal, but what if these were a map (or tuple list) from `Operator` to `BuiltinDefinition`